moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1496,19 +1496,25 @@ bool RobotState::getJacobian(const JointModelGroup* group, const LinkModel* link
   const int rows = use_quaternion_representation ? 7 : 6;
   const int columns = group->getVariableCount();
```

Comment:
ah! columns should not be the GROUP's variable count. Should be the entire robot?